js/components/dashboard/dashboard.js

```diff
@@ -34,6 +36,12 @@ export default class Dashboard extends PureComponent {
     this.activityHeaders.addEventListener('scroll', this.onHeadersScroll)
     this.onResize()
   }
+  componentDidUpdate(prevProps) {
```

Comment:
I am not convinced we need to do anything special here in `componentDidUpdate`.  

I didn't see any glitchy or weird behavior without it. 

I leave it to @dstrawberrygirl and @pjanik to figure out the best path if anything needs to be done.